onnx/shape_inference/implementation.h

```diff
@@ -39,6 +41,11 @@ class GraphInferencerImpl : public GraphInferencer {
       const std::vector<const TypeProto*>& inputTypes,
       const std::vector<const TensorProto*>& inputData) override;
 
+  SymbolTable& getSymbolTable() {
+    auto* contextForSymbol = const_cast<GraphInferenceContext*>(context_);
+    return contextForSymbol->symbolTable;
+  }
+
  private:
   GraphProto* g_;
   const GraphInferenceContext* context_;
```

Comment:
I think we should remove the const qualifier... making context_ const suggests that we do not change anything but we do... we do update the symboltable ...

Reply:
Good point. I have made GraphInferenceContext non-const